src/ImageProcessorCore/Colors/ColorTransforms.cs

```diff
@@ -18,9 +19,60 @@ namespace ImageProcessorCore
     public partial struct Color
     {
         /// <summary>
+        /// Blends two colors by lightening.
+        /// </summary>
+        /// <remarks>
+        /// Selects the lighter of the backdrop and source colors.
+        /// The backdrop is replaced with the source where the backdrop is lighter; otherwise, it is left unchanged
+        /// Note: This behavior is on a channel by channel basis, i.e., this rule is applied to each of the 3 RGB color channels separately.
+        /// </remarks>
+        /// <param name="backdrop"></param>
+        /// <param name="source"></param>
+        /// <returns>
+        /// The <see cref="Color"/>.
+        /// </returns>
+        public static Color Lighten(Color backdrop, Color source)
+        {
+            var newVector = new Vector4
```

Comment:
Since they are static methods on the `Color` struct they would be used in either scenario. Looking at the spec I would say it should take the higher of the two alpha values.